src/main/kotlin/com/jakewharton/dex/DexMember.kt

```diff
@@ -74,6 +81,8 @@ data class DexMethod(
 
   private companion object {
     val SYNTHETIC_SUFFIX = ".*?\\$\\d+".toRegex()
+    val LAMBDA_SIGNATURE = "lambda(\\$[a-zA-Z]+)*\\$\\d+(\\$[a-zA-Z]+)*".toRegex()
```

Comment:
Class and method names can include numbers and underscores. They can also include dollar signs, but we can probably ignore that.

Reply:
updated